### ISSUE TITLE
Need a better UI 

### ISSUE THREAD

**Issue by ZiaCodes** [ZiaCodes/counter.js#1]
Code a better UI for a better visualization and make sure it is in responsive across all the devices.

Frame work should be used - 
(1) ReactJS

### Create React App
```ssh
npx create-react-app my-app
cd my-app
npm start
```

### you can use any CSS library. 

if you do exactly same as below or better  , I will change the difficulty level to medium or maybe hard depend on your code & Design.. 

reference : 
<img src="https://user-images.githubusercontent.com/91361382/153700419-2bf69552-8fcb-420c-8b07-446b71d81703.png">

Before making pull request make sure you have done all the work and please add some necessary gifs and photos.

**Comment by manishbajagai2** [comment]
I want to work on this issue under JWOC @ZiaCodes

**Comment by manishbajagai2** [comment]
I would be happy if you provide a design for reference. Or i shall have a try on creating my own

**Comment by ZiaCodes** [comment]
> I would be happy if you provide a design for reference. Or i shall have a try on creating my own

Yes , You can try your design , make sure it is minimal and elegand

**Comment by manishbajagai2** [comment]
So am i being assigned with this issue?

**Comment by ZiaCodes** [comment]
> So am i being assigned with this issue?

Okay done

**Comment by ronitblenz** [comment]
Hi, I want to build the UI , Could you please assign me?

**Comment by ZiaCodes** [comment]
> Hi, I want to build the UI , Could you please assign me?

Only two issues at the time, complete then move on this one.

**Comment by r3yc0n1c** [comment]
Hey @ZiaCodes, I would like to work on this issue under JWoC'22 but I need more time for designing and putting everything all together. It would be really helpful if you upgrade the tag to `hard`.
Thanks.

**Comment by ZiaCodes** [comment]
> Hey @ZiaCodes, I would like to work on this issue under JWoC'22 but I need more time for designing and putting everything all together. It would be really helpful if you upgrade the tag to `hard`. Thanks.

Sure Sure, GO Ahead
